flask_sqlalchemy/__init__.py

```diff
@@ -403,6 +403,17 @@ def iter_pages(self, left_edge=2, left_current=2,
                 yield num
                 last = num
 
+    def as_dict(self):
+        """Create a dict with the pagination fields for use as a JSON response."""
```

Comment:
Need to mention that the result of this will probably not be directly JSON serializable. To serialize query results and models, the dev will need to implement a custom `Flask.json_encoder` class.